packages/vats/src/core/startWalletFactory.js

```diff
@@ -184,10 +184,12 @@ export const startWalletFactory = async (
     feeIssuerP,
   ]);
 
+  const poolBank = E(bankManager).getBankForAddress(poolAddr);
   const terms = await deeplyFulfilled(
     harden({
       agoricNames,
       board,
+      anyBank: poolBank,
```

Comment:
hm. This gives the walletFactory authority to _spend_ from the `poolBank`. That's not great. Gotta think that over...